build: bulk load build failed

## Bug Report

### 1. Describe the bug

<!-- A clear and concise description of what the bug is. -->

See https://github.com/tikv/migration/actions/runs/5113120222/jobs/9191966771?pr=342

```
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for TiKV Migration Project Parent POM 0.0.1-SNAPSHOT:
[INFO] 
[INFO] TiKV Migration Project Parent POM .................. FAILURE [ 15.660 s]
[INFO] Online Bulk Load ................................... SKIPPED
[INFO] Spark SST Data Source .............................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  15.882 s
[INFO] Finished at: 2023-05-29T15:04:50Z
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal on project tikv-migration-parent: Could not resolve dependencies for project org.tikv:tikv-migration-parent:pom:0.0.1-SNAPSHOT: Could not find artifact org.tikv:tikv-client-java:jar:3.3.0-SNAPSHOT -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
Error: Process completed with exit code 1.
```

### 2. Minimal reproduce step (Required)

<!-- a step by step guide for reproducing the bug. -->

### 3. What did you see instead (Required)

### 4. What did you expect to see? (Required)

### 5. What is your migration tool and TiKV version? (Required)

- TiKV:
- TiKV CDC:
- TiKV BR:
- TiKV Online Bulk Load: master(73d43788dfc2513d5bc3e579363438f147ccfb7d)

--- a/pom.xml
+++ b/pom.xml
@@ -122,7 +122,7 @@
         <dependency>
             <groupId>org.tikv</groupId>
             <artifactId>tikv-client-java</artifactId>
-            <version>3.3.0-SNAPSHOT</version>
+            <version>3.3.4</version>
         </dependency>
 
         <dependency>

